sbpy/units/core.py

```diff
@@ -205,37 +239,42 @@ def reflectance(wfb, cross_section=None, reflectance=None, **kwargs):
 
     Examples
     --------
-    Convertion between scattering cross-section and reflectance
     >>> import numpy as np
     >>> from astropy import units as u
-    >>> from sbpy.units import reflectance, VEGAmag, spectral_density_vega
-    >>> from sbpy.calib import solar_fluxd, vega_fluxd
+    >>> from sbpy.units import dimensionless_albedo
+    >>> from sbpy.units import VEGAmag, spectral_density_vega, albedo_unit
+    >>> from sbpy.calib import solar_fluxd
     >>>
     >>> solar_fluxd.set({'V': -26.77471503 * VEGAmag})
     ...                                             # doctest: +IGNORE_OUTPUT
-    >>> vega_fluxd.set({'V': 3.5885e-08 * u.Unit('W / (m2 um)')})
-    ...                                             # doctest: +IGNORE_OUTPUT
+
+    >>> # calculate albedo from magnitude
     >>> mag = 3.4 * VEGAmag
-    >>> cross_sec = np.pi * (460 * u.km)**2
-    >>> ref = mag.to('1/sr', reflectance('V', cross_section=cross_sec))
-    >>> print('{0:.4f}'.format(ref))
-    0.0287 1 / sr
-    >>> mag1 = ref.to(VEGAmag, reflectance('V', cross_section=cross_sec))
-    >>> print('{0:.2f}'.format(mag1))
-    3.40 mag(VEGA)
-
-    >>> # Convertion between magnitude and scattering cross-section
-    >>> ref = 0.0287 / u.sr
-    >>> cross_sec = mag.to('km2', reflectance('V', reflectance=ref))
-    >>> radius = np.sqrt(cross_sec/np.pi)
+    >>> xsec = np.pi * (460 * u.km)**2
+    >>> alb = mag.to(albedo_unit,
+    ...              dimensionless_albedo('V', cross_section=xsec))
+    >>> print('{0:.4f}'.format(alb))
+    0.0900 albedo
+
+    >>> # calculate size from magnitude
+    >>> alb = 0.09 * albedo_unit
+    >>> xsec = mag.to('km2', dimensionless_albedo('V', albedo=alb))
+    >>> radius = np.sqrt(xsec/np.pi)
     >>> print('{0:.2f}'.format(radius))
-    459.69 km
-    >>> mag2 = cross_sec.to(VEGAmag, reflectance('V', reflectance=ref))
-    >>> print('{0:.2f}'.format(mag2))
-    3.40 mag(VEGA)
-
+    460.11 km
+
+    >>> # calculate albedo from size
+    >>> radius = 460 * u.km
+    >>> xsec = np.pi * radius**2
+    >>> alb = xsec.to(albedo_unit, dimensionless_albedo('V', flux=mag))
+    >>> print('{0:.4f}'.format(alb))
+    0.0900 albedo
     """
```

Comment:
Do you have a good reference for more information on albedo and reflectance concepts?

Reply:
Yes, I will need to include references when we have a decision on the best approach.